CHANGELOG.md

```diff
@@ -1,3 +1,120 @@
+# 0.0.116rc1 - Jul 14, 2023 - "Anchoring the Roadmap"
```

Comment:
> Do you have any issue with the way its described in the first bullet point?

Yes, I would favor more a “big warning notice” about anchor outputs to say to our users this is the main thing we have worked for this release, and this is the one thing we would love to have API feedback on it.

Still while being conservative on funds risks coming with a beta feature. I had in mind what Dave did for mempoolfullrbf with 24.0.1: https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-24.0.1.md#notice-of-new-option-for-transaction-replacement-policies where the feature is explicitly said as conservative and there is not yet consensus between devs on the risks trade-offs.

> This is not true, the cfg flag has been removed as a part of "releasing anchors".

Ah that might be true, what is “releasing anchors” in this context the v0.0.116 rc0 ? Note it doesn’t show up in the GH https://github.com/lightningdevkit/rust-lightning/releases

> Do you have a specific suggestion that fits into the release notes, otherwise probably best to get it in the config docs (I think the docs there are pretty good, though, no?).

“Users are recommended to be conservative with the usage of `option_anchors_zero_fee_htlc_tx` and select their anchor outputs channels counterparty with care. With anchor output come the requiremeent a Lightning node has to manage “hot” fee-bumping reserves, a fundamental change in the operations of a node. Best practice would be to add progressively funds on this channel type until one is more familiar with the effective management of fee-bumping reserves. Further documentation is available around `negotiate_anchors_zero_fee_htlc_tx`”.

Feel free to tweak it according to your taste, though when a feature is new I think it’s better to say twice to our users a warning both in docs and in the release notes (and here I must say LND has the practice of doing it far better than us).

> That's news me - implementing reserves in a way that makes sense is super tricky, and we haven't figured out exactly how to do it after some discussion in https://github.com/lightningdevkit/rust-lightning/issues/2320. Marking that for the 117 milestone is a pretty big task given we are still in the design phase, and 117 already has a ton of things scheduled/tagged. Thus, I'm not sure we want to commit to it as a part of the next release.

Yeah sorry the “we” might not reflect the position of all of us, though at the very least I believe it reflect all the discussion over the anchor output patchetset (as tracked only recently with #https://github.com/lightningdevkit/rust-lightning/issues/2297) though I think I can say there is a “rough consensus” between you, Wilmer, Elias and me our fee-bumping reserves API can be improved.

For sure, “100% robust” fee-bumping reserves might be a too high step for 0.0.117 though I would be very happy if we can know clearly what set of PRs and substantial improvement of the fee-bumping reserves we aim with 0.0.117, and that way get us one step closer from marking anchor output “alpha” (somehow we shouldn’t delay too much anchor output once fee-bumping reserves API a bit mature as legacy channels have historical and known dust exposure).